teuthology/task/install/__init__.py

```diff
@@ -454,7 +454,12 @@ def task(ctx, config):
             priority: 1
             url: "http://location.repo"
 
-    Note: The 'repos' are supported for SUSE-based distros only, but patches
+          - name: "repo-alias"
+            gpgcheck = 1
+            gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
```

Comment:
This is invalid for yaml syntax.

Reply:
Yeah, the `dnf config-manager --add-repo` won't need this any more.